pkg/nodeidentity/azure/client.go

```diff
@@ -71,7 +71,7 @@ func newClient() (*client, error) {
 
 // getVMScaleSet returns the specified VM ScaleSet.
 func (c *client) getVMScaleSet(ctx context.Context, vmssName string) (compute.VirtualMachineScaleSet, error) {
-	return c.vmssesClient.Get(ctx, c.metadata.Compute.ResourceGroupName, vmssName)
+	return c.vmssesClient.Get(ctx, c.metadata.Compute.ResourceGroupName, vmssName, compute.UserData)
```

Comment:
I confirmed this is the only allowed value

https://github.com/Azure/azure-sdk-for-go/blob/b386adc15c2e60e3812956f68d244b608330c479/profiles/latest/compute/mgmt/compute/models.go#L256-L260